development/tools/cmd/synchronizer/tools/git.go

```diff
@@ -0,0 +1,62 @@
+package tools
```

Comment:
IMO the whole package should be moved to tools/pkg/synchronizer

Reply:
👍